include/renderer_gl/renderer_gl.hpp

```diff
@@ -44,8 +44,8 @@ class RendererGL final : public Renderer {
 	float oldDepthOffset = 0.0;
 	bool oldDepthmapEnable = false;
 
-	SurfaceCache<DepthBuffer, 10, true> depthBufferCache;
```

Comment:
Is there a particular use case that inspired going from 10 to 64?

Reply:
Had some crashes and bumping the number makes games more stable.

Reply:
Try configuring CMake with -DDISABLE_PANIC_DEV=ON
This will make these games not crash and use a ring buffer instead

Reply:
(Prolly need to delete CMakeCache first)

Reply:
Should I keep the bumped number or revert it back to 10?

Reply:
I'll bump it down for now and consider more carefully how to adjust it in the guture should the need arise